app/controllers/concerns/foreman_openscap/hosts_and_hostgroups_extensions.rb

```diff
@@ -0,0 +1,23 @@
+module ForemanOpenscap
+  module HostsAndHostgroupsExtensions
```

Comment:
The module name suggests that it extends hosts and hostgroups, but it extends the controllers, so it should be `HostsAndHostgroupsControllerExtensions`.

Reply:
Fixed!